m4_sessions/1810-inning/problems_tech_cannot_solve.md

```diff
@@ -21,11 +20,11 @@
 (Please include who is facilitating and how much time you are going to spend in each section):
 
 ## Recommended Trajectory 
-
-    ## Introductons 
-    ## Intro to new material
-    ## Small Group Activity
-    ## Closing Discussion
+  *  Introductons 10mins
+  *  Intro to several specific topics. We all introduce the topics 10mins.
+  *  Small Group debate/discussion groups of 4.  2 for argument, 2 against argument.  Switch topics and groups half  way through 20mins 
+  *  Closing Discussion as a group to review key takeaways.  We all talk about key take aways. 15 mins. 
```

Comment:
Be clear on discussion questions you will ask and come up with YOUR list of take aways so that you can be sure that students are getting the main ideas.